server/events/vcs/azuredevops_client.go

```diff
@@ -257,19 +257,19 @@ func (g *AzureDevopsClient) UpdateStatus(ctx context.Context, request types.Upda
 	opts := azuredevops.PullRequestListOptions{}
 	source, resp, err := g.Client.PullRequests.Get(g.ctx, owner, project, request.PullNum, &opts)
 	if err != nil {
-		return errors.Wrap(err, "getting pull request")
+		return "", errors.Wrap(err, "getting pull request")
```

Comment:
is this blank because there's no "status id" for azure statuses? Or because you didn't implement it? In these situations, I'd like for us to add TODO's or a comment of some sort otherwise, how would we know that we need to do this in the future?  Same goes for the other providers.